financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthScreen.kt

```diff
@@ -163,7 +167,7 @@ private fun LoadedContent(
     onSelectAnotherBank: () -> Unit
 ) {
     when (authenticationStatus) {
-        is Uninitialized -> when (payload.authSession.isOAuth ?: false) {
+        is Uninitialized -> when (payload.authSession.isOAuth) {
```

Comment:
why this change?

Reply:
isOAuth is not nullable anymore, this was left when it used to be null and IDE complained this time